stdlib/grpc/src/main/resources/templates/skeleton/sampleService.mustache

```diff
@@ -0,0 +1,47 @@
+import ballerina/grpc;
+import ballerina/io;
+
+endpoint grpc:Listener listener {
+    host:"hostname",
+    port:9000
+};
+{{#if bidiStreaming}}
+@grpc:ServiceConfig { {{#streamingFunction}}
+    name: {{methodName}},{{/streamingFunction}}
+    clientStreaming: true,
+    serverStreaming: true
+}{{/if}}{{#if clientStreaming}}
+@grpc:ServiceConfig { {{#streamingFunction}}
+    name: {{methodName}},{{/streamingFunction}}
+    clientStreaming: true
+}
+{{/if}}{{#if unary}}
+@grpc:ServiceConfig
+{{/if}}
+service {{serviceName}} bind listener {
+{{#streamingFunction}}
+    onOpen(endpoint caller) {
+        // Implementation goes here.
+    }
+
+    onMessage(endpoint caller, {{inputType}} value) {
+        // Implementation goes here.
+    }
+
+    onError(endpoint caller, error err) {
+        // Implementation goes here.
+    }
+
+    onComplete(endpoint caller) {
+            // Implementation goes here.
+            // You should return a {{outputType}}
```

Comment:
From the RPC concept the client is calling a method and expecting a return value. Not like in http.

Reply:
What I meant is that since return concept is not there for resources a better wording would be nice. Let's leave it if `return` would make more sense to a GRPC user.